dev-libs/libsigc++/libsigc++-2.99.7.recipe

```diff
@@ -55,12 +54,13 @@ INSTALL()
 {
 	make install
 
-	prepareInstalledDevelLib libsigc-2.0
+	prepareInstalledDevelLib \
+		libsigc-3.0
```

Comment:
>  i just tried your proposed fix, but it doesn't help much.

I guess you are talking about the git commands. Does the command with the **`git remote -v`** yeld what I was expecting? If not, could you run:
```
cd /path/to/haikuports
git remote -v | grep haikuports/haikuports
```
and copy/paste the result here? (I just need the two lines of output).
I think you'll get this:
```
real_origin        https://github.com/haikuports/haikuports.git (fetch)
real_origin        https://github.com/haikuports/haikuports.git (push)
```
It should be 2 lines which both start with the same word, e.g. "real_origin". That word is what you should use instead of the word "upstream" in the commands of my first message in this PR.

But don't worry if you are lost. We can always do as follows:
a) I grab your remote branch, update it localy, do the rebase against current master and then push it back to your remote branch;
b) then you delete your local branch and grab a fresh copy of your newly updated (by me) remote branch.
So if you want to try this, just let me know and as soon as I have finished step a) I'll send you the commands that you should type for step b).

Of course, you'll have to address @korli's comment. (Later I can try to help you with that too if you wish.)